tests/estimation/test_ols_mediation.py

```diff
@@ -0,0 +1,63 @@
+"""
+benchmark_mediation.py::ols_mediation
+
+We test :
+- Coefficient product estimator for a certain tolerance
+- Whether total effect = direct + indirect
+"""
+
+from pprint import pprint
+import pytest
+import numpy as np
+from numpy.random import default_rng
+from med_bench.src.get_simulated_data import simulate_data
+from med_bench.src.benchmark_mediation import ols_mediation
+
+
+ATE_TOLERANCE = 0.05
+DIRECT_TOELERANCE = 0.05
+INDIRECT_TOLERANCE = 0.05
+TOLERANCE = [
+    ATE_TOLERANCE,
+    DIRECT_TOELERANCE,
+    DIRECT_TOELERANCE,
+    INDIRECT_TOLERANCE,
+    INDIRECT_TOLERANCE,
+]
+
+
+data = simulate_data(
```

Comment:
make similar changes. Maybe need to refactor and make a common simulation fixture. DRY framework